onnx/reference/ops/aionnx_preview_training/_op_list.py

```diff
@@ -4,28 +4,24 @@
 from __future__ import annotations
 
 import textwrap
-from typing import Any, Dict
-from typing import Optional as TOptional
-from typing import Union
+from typing import Any
 
-from onnx.reference.op_run import OpFunction
+from onnx.reference.op_run import OpFunction, OpRun
 from onnx.reference.ops._helpers import build_registered_operators_any_domain
-from onnx.reference.ops.aionnx_preview_training._op_run_training import OpRunTraining
 from onnx.reference.ops.aionnx_preview_training.op_adagrad import Adagrad
 from onnx.reference.ops.aionnx_preview_training.op_adam import Adam
 from onnx.reference.ops.aionnx_preview_training.op_momentum import Momentum
 
 
-def _build_registered_operators() -> dict[str, dict[int | None, OpRunTraining]]:
-    return build_registered_operators_any_domain(globals().copy())  # type: ignore[return-value]
+def _build_registered_operators() -> dict[str, dict[int | None, type[OpRun]]]:
```

Comment:
Why not use OpRunTraining?

Reply:
Because type checking doesn't allow overriding parent type with sub type.